spec/text-editor-component-spec.coffee

```diff
@@ -77,6 +82,37 @@ describe "TextEditorComponent", ->
 
       expect(component.lineNodeForScreenRow(0).textContent).not.toBe("You shouldn't see this update.")
 
+  describe "measurements", ->
+    it "is equivalent to TextEditorPresenter::pixelPositionForScreenPosition", ->
```

Comment:
I like this comparison test for now, but will we be able to drop the old method before merging this and test the new method directly?